packages/backend/src/tokens/types.ts

```diff
@@ -49,6 +50,11 @@ export type AuthenticateRequestOptions = {
    * @internal
    */
   apiClient?: ApiClient;
+  /**
+   * The type of token to accept.
+   * @default 'session_token'
+   */
+  acceptsToken?: TokenType | TokenType[] | 'any';
```

Comment:
I personally prefer APIs where there is no mix between string and array of strings.

So default would be `['session_token']`. Then I don't need to wrap it with an array when I quickly want to switch to two token types.